frontend/src/App.vue

```diff
@@ -150,6 +156,7 @@ const routeHasModal = computed(
 const getDbData = async () => {
   if (currentUser?.exists()) {
     await Promise.all([
+      userStore.profile(call),
```

Comment:
This fixes the issue where the FTUE keeps showing up even though they're done. It's also good to fetch an updated copy more often than we do.